inz-demo/description.xml

```diff
@@ -1,18 +1,18 @@
 <SchemeManager version="7">
 	<Id>inz-demo</Id>
-	<Url>http://localhost:7000/</Url>
+	<Url>https://irma-scheme.local.internetnz.nz</Url>
 	<Demo>true</Demo>
 	<Name>
-		<en>INZ Demo</en>
-		<nl>INZ Demo</nl>
+		<en>INZ Scheme Manager Demo</en>
+		<nl>INZ Scheme Manager Demo</nl>
 	</Name>
 	<Description>
 		<en>Demo credentials within the INZ domain</en>
 		<nl>Demo INZ-credentials</nl>
 	</Description>
 	<KeyshareServer>http://localhost:18080/irma_keyshare_server/api/v1</KeyshareServer>
```

Comment:
the current port here is 8080 unless you change the keyshare server as well.

Reply:
Maybe I will change later when integrating with keyshare. In this PR I only need to focus on the scheme changes.
18080 is a port using a proxy on local. I need to stick with it, for now, just to capture the requests.